### PR TITLE
[Admin] Edit, delete category

### PR DESCRIPTION
**Pull request by ChuThom610**
- Link redmine: https://edu-redmine.framgia.vn/issues/1428

### CODE REVIEW

**Comment by ChuThom610** [comment]
ready

File: app/controllers/admin/categories_controller.rb [hunk 1]
```diff
@@ -34,7 +34,7 @@ def destroy
     else
       flash[:danger] = t "flashs.fail_delete_category"
     end
-    redirect_to request.referrer || categories_url
+    redirect_to categories_url
```
Comment (huongnguyenmta) [inline app/controllers/admin/categories_controller.rb]: trong admin mà nó ra trang kia luôn hở em?

**Comment by huongnguyenmta** [comment]
commented

**Comment by ChuThom610** [comment]
ready

**Pull request merged**